components/director/internal/domain/bundle/fixtures_test.go

```diff
@@ -397,6 +399,10 @@ func fixEntityBundle(id, name, desc string) *bundle.Entity {
 		Links:                         repo.NewValidNullableString("[]"),
 		Labels:                        repo.NewValidNullableString("[]"),
 		CredentialExchangeStrategies:  repo.NewValidNullableString("[]"),
+		CorrelationIDs: sql.NullString{
```

Comment:
Isn't that cleaner:

`CorrelationIDs:  repo.NewValidNullableString(correlationIDs)`